### DISCUSSION TITLE
After the browser update, the right-click menu is not working.

### DISCUSSION THREAD

**Discussion by meteormit** [nextai-translator/nextai-translator#1702] [Q&A] [answered]
(Regarding Browser Extension)
After the update of the Edge browser, the mini features for text selection have disappeared. Although the options in the right-click menu are still there, they do not respond when clicked. Is this a settings issue or a problem with the Edge update?

**Comment by myyeti1043** [comment]
Chrome also encountered a similar issue, plugin version 0.4.32, Chrome version 130.0.6723.70.

**Reply by myyeti1043** [reply, marked as answer]
Someone solved this problem in #1701.

**Reply by meteormit** [reply]
Thank you for your guidance!
